impact.py

```diff
@@ -7,6 +7,35 @@
 import time
 
 
+def construct_weather(paths, prednames, metadata):
```

Comment:
another way to do this would be to accept prename, path kwargs, e.g.

    def construct_weather(**weather):
        prednames = weather.keys()
        weather_data = []
        for predname in prednames:
            with xr.open_dataset(weather[predname]) as ds:
                weather_data.append(ds.load())
        return xr.concat(weather_data, dim=pd.Index(prednames, name='prednames'))

You could call this like this:

    weather = construct_weather(**{'tas': '/path/to/tas.nc', 'tas-poly-2': '/path/to/tas-poly-2.nc', ...})

This further frees people up to call this function however they want (they don't have to be in a jrnr-like environment).

Reply:
dude this is awesome

Reply:
thanks! 🎉

Reply:
but won't his return a dataset

Reply:
should be `weather_data.append(ds[pred].load())`, no?